From a91093e7e85254cbb6564ea7b795402e2f1b3c9c Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 20 Jun 2024 09:43:47 +0200
Subject: [PATCH 32/65] misc: stm32: use reset driver name instead of default
 name

In case of error, display the real reset driver's name instead of
default string "stm32_rcc_reset".

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git d9ed7e96f1f6b7112ee79303a5bb31989beeb6d2 misc: stm32: use reset driver name instead of default name]
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: I848e061b65155b8e632cb10dc44ce24cb06b7a0d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/388693
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/misc/stm32_rcc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/stm32_rcc.c b/drivers/misc/stm32_rcc.c
index 15bf69f7d57..752f10e368a 100644
--- a/drivers/misc/stm32_rcc.c
+++ b/drivers/misc/stm32_rcc.c
@@ -76,7 +76,7 @@ static int stm32_rcc_bind(struct udevice *dev)
 
 	drv = lists_driver_lookup_name(rcc_clk->drv_name_rst);
 	if (!drv) {
-		dev_err(dev, "Cannot find driver stm32_rcc_reset'\n");
+		dev_err(dev, "Cannot find driver '%s'\n", rcc_clk->drv_name_rst);
 		return -ENOENT;
 	}
 
-- 
2.51.0

